### DISCUSSION TITLE
🦀 Rust compiler now required to install terra from source

### DISCUSSION THREAD

**Discussion by javabster** [Qiskit/.github#23] [Announcements]
:rotating_light: Announcement! :rotating_light:

With the merging of [#7658](https://github.com/Qiskit/qiskit-terra/pull/7658) contributors to Qiskit will now need to have a Rust compiler installed in order to install Qiskit Terra from source.

To update your dev environment you can check out the updated [Contributing Guidelines](https://github.com/Qiskit/qiskit-terra/blob/main/CONTRIBUTING.md#installing-qiskit-terra-from-source) or follow the instructions in the [rust documentation](https://rustup.rs/) on how to install the compiler.

**Note:** don't forget you may need to restart your shell or update your current session according to the rust docs:
```
To get started you may need to restart your current shell. 
This would reload your PATH environment variable to include Cargo's bin directory ($HOME/.cargo/bin).

To configure your current shell, run: source $HOME/.cargo/env
```

**Comment by jwoehr** [comment]
On Ubuntu(-like) Linux distros, install Rust via `apt install cargo`

**Reply by Raghav-Bell** [reply]
Thanks @jwoehr 
Here are docs for more details for installation of Rust on different platforms:
[Other Rust Installation Methods](https://forge.rust-lang.org/infra/other-installation-methods.html#other-rust-installation-methods)
Thanks
